app/metrics.yaml

```diff
@@ -5892,6 +5910,25 @@ pocket:
     metadata:
       tags:
         - PocketIntegration
+  spoc_shown:
```

Comment:
I'd recommend structuring this slightly differently: let's just have one metric, `spoc_shim` and add two "reasons" to the ping: "impression" and "click". This should make analysis a bit clearer, so that we don't need to think about nulls.